ros/src/waypoint_updater/cfg/DynReconf.cfg

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+PACKAGE = "waypoint_updater"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+gen.add("dyn_update_rate",    int_t,    0, "An Integer parameter", 10,  1, 25)
```

Comment:
Can we increase the max update rate from 25 to 50 Hz to give us the full window up to match DBW node?

Reply:
Moved up to 50